classes/class-bsf-custom-fonts-render.php

```diff
@@ -162,7 +164,7 @@ public function add_style() {
 				}
 				?>
 				<style type="text/css">
-					<?php echo strip_tags( $this->font_css ); ?>
+					<?php echo wp_kses_post( wp_strip_all_tags( $this->font_css ) ); ?>
```

Comment:
`wp_kses_post` - we can replace this with `esc_html()` ?

Reply:
Done